tuning/sft_trainer.py

```diff
@@ -113,6 +67,11 @@ def train(
             The peft configuration to pass to trainer
         trainer_control_args: configs.TrainerControllerArguments \
             for controlling the training loop using policy rules
+        additional_callbacks: List of callbacks to attach with SFTtrainer , besides those associated with experiment trackers or TrainerControllers. Callbacks associated with tracker with automatically be added.
+        trackers: List of the available trackers in trackers.tracker_factory.REGISTERED_TRACKERS
```

Comment:
there doesnt seem to be an argument `trackers` anymore

Reply:
fixed